### PR TITLE
feat: allow to setup prometheus integration

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR allows to setup promethus integration for monitoring Flower.

### Screenshots
image

### CODE REVIEW

File: tutorcelery/plugin.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
         ("CELERY_FLOWER_HOST", "flower.{{LMS_HOST}}"),
         ("CELERY_FLOWER_DOCKER_IMAGE", "docker.io/mher/flower:2.0.1"),
         ("CELERY_MULTIQUEUE_ENABLED", False),
+        ("CELERY_FLOWER_PROMETHEUS", True),
```
Comment (jfavellar90) [inline tutorcelery/plugin.py]:
```suggestion
        ("CELERY_FLOWER_SERVICE_MONITOR", True),
```

Comment (jfavellar90) [reply tutorcelery/plugin.py]: This one should be `False` by default?

File: tutorcelery/patches/k8s-deployments [hunk 2]
```diff
@@ -84,4 +84,30 @@ spec:
             value: "5555"
           - name: FLOWER_BASIC_AUTH
             value: {{CELERY_FLOWER_BASIC_AUTH}}
+
+{% if CELERY_FLOWER_SERVICE_MONITOR -%}
```
Comment (jfavellar90) [inline tutorcelery/patches/k8s-deployments:87]:
```suggestion
{% if CELERY_FLOWER and CELERY_FLOWER_SERVICE_MONITOR -%}
```

Comment (Ian2012) [reply tutorcelery/patches/k8s-deployments:87]: This if statement is nested on a `{% if CELERY_FLOWER %}` statement already

File: tutorcelery/patches/k8s-deployments [hunk 3]
```diff
@@ -84,4 +84,30 @@ spec:
             value: "5555"
           - name: FLOWER_BASIC_AUTH
             value: {{CELERY_FLOWER_BASIC_AUTH}}
+
```
Comment (jfavellar90) [inline tutorcelery/patches/k8s-deployments]: No need this extra space

File: tutorcelery/patches/k8s-deployments [hunk 4]
```diff
@@ -84,4 +84,30 @@ spec:
...
+    path: /metrics
+    schema: http
+  namespaceSelector:
+    matchNames:
+    - {{ K8S_NAMESPACE }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: flower
+{%- endif %}
+
```
Comment (jfavellar90) [inline tutorcelery/patches/k8s-deployments]: No need this extra space

**Review by jfavellar90 — APPROVED** [review]

**Comment by jfavellar90** [comment]
Left comments for quick fixes, the rest LGTM

**Pull request merged**